src/AuthKeyValueUser.php

```diff
@@ -247,12 +239,19 @@ public function getPropertyValues($property)
                     $returnValue = array($value);
                 }
             }
+
+            if (!empty($returnValue)) {
+                $returnValue = json_decode(current($returnValue), true);
+
+                if (!is_array($returnValue)) {
+                    $returnValue = [$returnValue];
+                }
+            }
```

Comment:
Are we 100% sure, that we shouldn't have empty property values?

Reply:
I'm not 100% sure on anything aside my name :) Sometimes it is was null (as on the docblock) tha's why I had to check.